hydra-node/test/Hydra/Model/Payment.hs

```diff
@@ -86,6 +86,7 @@ instance IsTx Payment where
   type UTxOType Payment = [(CardanoSigningKey, Value)]
   type ValueType Payment = Value
   txId = error "undefined"
+  txSpendingUTxO = error "undefined"
```

Comment:
Will the CI now error out now because of the warning that this is not implemented?

Reply:
No if it wasn't implemented at all before I don't think it it used anywhere in the tests.

Reply:
Having this error out would be a good litmus test whether the `-Werror` CI actually caught this one. Was this the case?

Reply:
Yes it's caught by nix flake check